service_auth/app/models/Group.scala

```diff
@@ -53,15 +57,17 @@ object GroupAdd {
 
   implicit def reads: Reads[GroupAdd] = Json.reads[GroupAdd]
 
-  def applyForm(name: String, permInt: Int, mid: Int, maybeDesc: Option[String]) =
-    GroupAdd(name, Permission.fromInt(permInt), MuseumId(mid), maybeDesc)
+  def applyForm(name: String, moduleId: Int, permInt: Int, mid: Int, maybeDesc: Option[String]) =
+    GroupAdd(name, GroupModule.unsafeFromInt(moduleId), Permission.fromInt(permInt), MuseumId(mid), maybeDesc)
 
-  def unapplyForm(g: GroupAdd) =
-    Some((g.name, g.permission.priority, g.museumId.underlying, g.description))
+  def unapplyForm(g: GroupAdd) = Some(
+    (g.name, g.module.id, g.permission.priority, g.museumId.underlying, g.description)
+  )
 
   val groupAddForm = Form(
     mapping(
       "name"        -> text(minLength = 3),
+      "module"      -> number.verifying(GroupModule.fromInt(_).nonEmpty),
```

Comment:
This is now fixed, by adding an extra dropdown in the backend interface for group management